app/controllers/api/v1/massages_controller.rb

```diff
@@ -0,0 +1,9 @@
+# app/controllers/api/massages_controller.rb
+module Api
+  class Api::V1::MassagesController < ApplicationController
+    def random_massage
+      @greeting = Message.order('RANDOM()').first
+      render json: { message: @greeting.message }
+    end
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update the way you render data from your api end point in the it also displays the `created_at` and `update_at` attribute date time. e.g.: `{"id":4,"message":"bojr","created_at":"2023-04-06T00:28:41.571Z","updated_at":"2023-04-06T00:28:41.571Z"}`. You can only change it to some thing like this:
```ruby
  def index
      @greeting = Message.order('RANDOM()').first
      render json: @greeting
    end
```